custom-metrics-stackdriver-adapter/adapter-beta.yaml

```diff
@@ -49,7 +49,7 @@ subjects:
   name: custom-metrics-stackdriver-adapter
   namespace: custom-metrics
 ---
-apiVersion: extensions/v1beta1
```

Comment:
nit: in general please put the changes that are not related to dependency update in other commit for easier review.